repairnator/docker-images/pipeline-dockerimage/pipeline_launcher.sh

```diff
@@ -9,8 +9,8 @@ function ca {
     echo "$1 $2 "
   fi
 }
-
-args="`ca --dbhost $MONGODB_HOST``ca --dbname $MONGODB_NAME``ca --pushurl $PUSH_URL``ca --smtpServer $SMTP_SERVER``ca --smtpPort $SMTP_PORT``ca --smtpUsername $SMTP_USERNAME``ca --smtpPassword $SMTP_PASSWORD``ca --notifyto $NOTIFY_TO``ca --githubUserName $GITHUB_USERNAME``ca --githubUserEmail $GITHUB_USEREMAIL``ca --experimentalPluginRepoList $EXPERIMENTAL_PLUGIN_REPOS`"
+echo $SMTP_PORT
```

Comment:
I would suggest to print the whole sentence, or remove this line.

Something like `echo "Repairnator-pipeline is using mail server $SMTP_SERVER:$SMTP_PORT to send notifications"`

Reply:
nice suggestion . Sure :) , I'll do that when we are done with the CI problem .

Reply:
hi @gluckzhang :) , I'll just remove the `echo SMTP_PORT` since you'll get the same information after launching the pipeline as below provided by `repainator.jar` . 
```
23:32:57.353 [main] INFO  f.i.s.repairnator.pipeline.Launcher - Start by getting the build (buildId: 1) with the following config: RepairnatorConfig{runId='1', launcherMode=REPAIR, inputPath='null', outputPath='.', mongodbHost='null', mongodbName='repairnator', smtpServer='null', smtpPort='25', smtpTLS='true', smtpUsername='null', smtpPassword='', notifyTo=[], notifyEndProcess=false, push=false, pushRemoteRepo='null', fork=false, lookFromDate=null, lookToDate=null, buildId=1, z3solverPath='./z3_for_linux', workspacePath='./workspace', githubToken='invalid_to[...]', dockerImageName='null', skipDelete=false, createOutputDir=false, logDirectory='null', nbThreads=1, globalTimeout=0, whiteList=null, blackList=null, jobSleepTime=0, buildSleepTime=0, maxInspectedBuilds=0, duration=null, humanPatch=false, repository='null', clean=true, bearsMode=BOTH, bearsDelimiter = false, repairTools=NPEFix, githubUserName= repairnator, githubUserEmail=noreply@github.com, pipelineMode=null, listenerMode=NOOP, activeMQUrl=tcp://localhost:61616, activeMQSubmitQueueName=null}
```